### ISSUE TITLE
Wrong checksum for v1.4.0

### ISSUE THREAD

**Issue by r3nic1e** [SquadcastHub/terraform-provider-squadcast#48]
After upgrading to the latest version of the provider terraform cannot validate squadcast provider checksum:

```
❯ terraform init

Initializing the backend...
Initializing modules...

Initializing provider plugins...
- Finding squadcasthub/squadcast versions matching "~> 1.3, ~> 1.4"...
- Finding hashicorp/random versions matching "~> 3.4"...
- Installing squadcasthub/squadcast v1.4.0...
- Installing hashicorp/random v3.5.1...
- Installed hashicorp/random v3.5.1 (signed by HashiCorp)
╷
│ Error: Failed to install provider
│
│ Error while installing squadcasthub/squadcast v1.4.0: checksum list has unexpected SHA-256 hash afa8e9618187625a115ce0f8314061fcf215053e7e77c2d83e0258fa0b852280 (expected
│ 2ad75de123269ff887f2328bd9bb61dd566e4e970c7f098b50574738a4a45da5)
╵

❯ terraform version
Terraform v1.5.1
on darwin_amd64
```

**Comment by mohammadmuazam** [comment]
We are taking a look at this.

**Comment by SandraGarciaSerrano** [comment]
Same issue here

**Comment by roshan8** [comment]
We have released terraform version 1.4.2. Can you please try that? @SandraGarciaSerrano @r3nic1e

**Comment by r3nic1e** [comment]
@roshan8 Thanks, v1.4.2 works
